docker-compose/config/pm2-dev.json

```diff
@@ -3,8 +3,8 @@
     "name"         : "KuzzleServer",
     "script"       : "bin/kuzzle",
     "args"         : ["start"],
-    "watch"        : ["lib", "config", "bin", "default.config.js"],
-    "ignore_watch" : ["node_modules"],
-    "node_args": "--inspect"
+    "watch"        : ["lib", "config", "bin", "default.config.js", "plugins/enabled"],
```

Comment:
Hm. But what happens if a plugin use the filesystem for, say, lock a resource, open a unix socket, or log things in a file?

I'm more in favor of watching only .js files